packages/h5p-server/test/H5PPlayer.renderHtmlPage.test.ts

```diff
@@ -39,7 +39,7 @@ describe('Rendering the HTML page', () => {
                     <script>
                         window.H5PIntegration = {
                   "ajax":{
-                     "contentUserData":"/h5p/contentUserData",
+                     "contentUserData":"/h5p/contentUserData/:contentId/:dataType/:subContentId",
```

Comment:
We should consider moving to Jest Snapshots. Mainting this test file is quite annoying. I have touched it myself several times and it's very redundant work.

Reply:
Ok, I made an issue for that: https://github.com/Lumieducation/H5P-Nodejs-library/issues/1892